ethcore/evm/src/interpreter/mod.rs

```diff
@@ -525,6 +543,32 @@ impl<Cost: CostType> Interpreter<Cost> {
 			instructions::JUMPDEST => {
 				// ignore
 			},
+			instructions::BEGINSUB => {
+				// ignore
+			},
+			instructions::JUMPSUB => {
+				if self.return_stack.len() > 1023 {
```

Comment:
it the previous PR #11612 this was a configurable parameter, but I think we should use a global constant